src/main/kotlin/se/uulm/snowballr/backend/model/plugins/FetcherPaper.kt

```diff
@@ -0,0 +1,84 @@
+package se.uulm.snowballr.backend.model.plugins
+
+import kotlinx.serialization.Serializable
+import se.uulm.snowballr.backend.model.dto.Author
+import se.uulm.snowballr.backend.model.dto.Paper
+import java.time.OffsetDateTime
+import java.util.UUID
+
+/**
+ * Represents an author fetched from an external source.
+ *
+ * @property firstName The first name of the author.
+ * @property lastName The last name of the author.
+ * @property orcid The ORCID of the author, if available.
+ */
+@Serializable
+data class FetcherAuthor(
+    val firstName: String,
+    val lastName: String,
+    val orcid: String? = null,
+)
+
+/**
+ * Represents a paper fetched from an external source.
+ *
+ * @property title The title of the paper.
+ * @property externalId A unique external identifier for the paper, such as a DOI or other unique ID.
+ * @property abstract The abstract or summary of the paper.
+ * @property year The year the paper was published in.
+ * @property publisher The name of the publisher of the paper.
+ * @property publicationType The type of publication, such as "journal", "conference", etc. (unconstrained)
+ * @property publicationName The name of the publication where the paper appeared.
+ * @property fetcherMetadata A map of metadata used by fetchers. For example, API-related unique IDs for later use.
+ * @property authors A set of authors associated with the paper, represented as [FetcherAuthor] objects.
+ */
+@Serializable
+data class FetcherPaper(
+    val title: String,
+    val externalId: String? = null,
+    val abstract: String,
+    val year: Int,
+    val publisher: String,
+    val publicationType: String,
+    val publicationName: String,
+    val fetcherMetadata: Map<String, String> = emptyMap(),
+    val authors: Set<FetcherAuthor> = emptySet(),
+)
+
+/**
+ * Extension function to convert a FetcherAuthor object into an Author object to be inserted into the database.
+ *
+ * @receiver FetcherAuthor The FetcherAuthor instance to be converted.
+ * @return Author The resulting Author.
+ */
```

Comment:
```suggestion
/**
 * Converts a [FetcherAuthor] into an [Author] object to be inserted into the database.
 *
 * @receiver The [FetcherAuthor] instance to be converted.
 * @return The resulting [Author] dto.
 */
```
Same style please for `toPaper()`